packages/vue-ds/src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <nav :class="breadcrumbClass" aria-label="breadcrumb">
+    <slot />
```

Comment:
here we will accept any component, or just the `BreadcrumbItem` ? 🤔

Reply:
Just `BreadcrumbItem`, have you any idea to use just `BreadcrumbItem` as component here?

Reply:
just a curiosity hahahahahah Vue have the things very well defined, that I thought I could only allow one type of element to be passed as a slot 😂